cylc/flow/hostuserutil.py

```diff
@@ -113,7 +113,7 @@ def _get_host_info(self, target=None):
         """Return the extended info of the current host."""
         if target not in self._host_exs:
             if target is None:
-                target = socket.getfqdn()
+                target = socket.gethostname()
             try:
```

Comment:
What's the deal with this change?

Reply:
Whoops.

BTW this is the change which makes Cylc work on latest Mac OS.

Reply:
Ah, good to know.